src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
+  } else if (days >= 3) {
+    result = result - 20;
```

Comment:
Replace the hard-coded value `3` with a constant named `SHORT_TERM_RENTAL_DAYS` or similar, and replace `20` with a constant like `SHORT_TERM_DISCOUNT`. Also, remove the `else` keyword as it's redundant when the `if` branch contains a `return` statement.